README.md

```diff
@@ -5,7 +5,7 @@ A Prisma engine adaptation for React Native. Please note that this is a [Preview
 ## Installation
 
 ```
-npm i --save --save-exact @prisma/client@5.14.0-dev.26 @prisma/react-native@5.14.0-dev.26 react-native-quick-base64
+npm i --save --save-exact @prisma/client@5.14.0-dev.67 @prisma/react-native@5.14.0-dev.67 react-native-quick-base64
```

Comment:
I think we can use `^5.14.0` version rather than picking specific dev snapshots now.

Reply:
Ok, to confirm, shall we `@latest` or `^5.14.0`?

Reply:
just did latest